copyright/copyright.js

```diff
@@ -0,0 +1,106 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+const path = require('path')
+
+const red = '\x1b[31m'
+const green = '\x1b[32m'
+const yellow = '\x1b[33m'
+const magenta = '\x1b[35m'
+const cyan = '\x1b[36m'
+const blackBG = '\x1b[40m'
+const defaultBG = '\x1b[49m'
+const defaultFG = '\x1b[39m'
+
+const args = process.argv.filter((arg) => {
+    return !/node|copyright/.test(arg)
+})
+
+const copyright = {
+    lintMode: true,
+    langs: {},
+    run() {
+        this.buildSupportedExtensions()
+        let files = []
+
+        const processedGlobs = args.map((folder) => {
+            files.push(glob.sync(folder))
+        })
```

Comment:
If you return `glob.sync(folder)` from the map function, then you won't need the `files` array; `processedGlobs` will contain what you want.